src/mcp_server_milvus/server.py

```diff
@@ -114,7 +119,7 @@ async def vector_search(
                 search_params=search_params,
                 limit=limit,
                 output_fields=output_fields,
-                filter=filter_expr,
+                filter_expr=filter_expr,
```

Comment:
I think the present code is correct. MilvusClient accepts filter as parameter, not filter_expr.